claragenomics/dl4atac/metrics.py

```diff
@@ -7,237 +7,515 @@
 # distribution of this software and related documentation without an express
 # license agreement from NVIDIA CORPORATION is strictly prohibited.
 #
+"""Metrics module."""
 
 import numpy as np
+
+from scipy.stats import rankdata
+
+import sklearn.metrics
+
 import torch
-import torch.distributed as dist
+
 import torch.nn.functional as F
-import sklearn.metrics
-from collections import Iterable
-from scipy.stats import rankdata
+
 
 class Metric(object):
```

Comment:
@avantikalal , please reivew the docstring for this entire file. I am not confident.